open-api/rest-catalog-open-api.yaml

```diff
@@ -313,6 +313,29 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+    head:
+      tags:
+        - Catalog API
+      summary: Check if a namespace exists
+      operationId: namespaceExists
+      description:
+        Check if a namespaces exists. This request does not return a response body.
```

Comment:
should this say that the response doesn't return a body instead of the request?

Reply:
updated to `Check if a namespace exists. This response does not return a body.`